alumnium/drivers/appium_driver.py

```diff
@@ -19,12 +20,16 @@
 
 
 class AppiumDriver(BaseDriver):
-    def __init__(self, driver: Remote):
+    def __init__(self, driver: Remote, mobile_platfrom: str):
```

Comment:
Likewise, we should determine the platform using `driver` variable instead of an explicit flag.